homeassistant/components/sensibo/coordinator.py

```diff
@@ -98,6 +98,30 @@ async def _async_update_data(self) -> dict[str, dict[str, Any]]:
             calibration_temp = dev["sensorsCalibration"].get("temperature", 0.0)
             calibration_hum = dev["sensorsCalibration"].get("humidity", 0.0)
 
+            # Sky plus supports functionality to use motion sensor as sensor for temp and humidity
+            if main_sensor := dev["mainMeasurementsSensor"]:
+                measurements = main_sensor["measurements"]
+                temperature = measurements.get("temperature")
+                humidity = measurements.get("humidity")
+
+            motion_sensors = []
+            if dev["motionSensors"]:
+                for motionsensor in dev["motionSensors"]:
+                    sensor = {
+                        "alive": motionsensor["connectionStatus"].get("isAlive"),
+                        "fw_ver": motionsensor.get("firmwareVersion"),
+                        "fw_type": motionsensor.get("firmwareType"),
+                        "id": motionsensor["id"],
+                        "ismainsensor": motionsensor.get("isMainSensor"),
+                        "batteryvoltage": motionsensor["measurements"].get(
```

Comment:
Please use _s to separate words.  This be easier to type later if you make it a dataclass

Reply:
Can you give me an example as I'm not entirely sure what you mean.